### PR TITLE
[SSLNTV-30] Add Contribution info to README

### PR DESCRIPTION
**Pull request by cam-rod**
https://issues.redhat.com/browse/SSLNTV-30

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,3 +132,10 @@ set "JAVA_HOME=%USERPROFILE%\apps\java-1.8.0"
 set "INCLUDE=%INCLUDE%;%OPENSSL_64%\include"
 mvn clean install
 ```
+
+Contributing
+============
+
+All new featurs and enhancements should be submitted to the main branch only. Our [contribution guide](https://github.com/wildfly-security/wildfly-openssl-natives/blob/main/CONTRIBUTING.md) will take you through the steps for getting started with the OpenSSL bindings, and how to format and submit your first PR.
```
Comment (fjuma) [inline README.md:139]: s/featurs/features :)

File: README.md [hunk 2]
```diff
@@ -132,3 +132,10 @@ set "JAVA_HOME=%USERPROFILE%\apps\java-1.8.0"
 set "INCLUDE=%INCLUDE%;%OPENSSL_64%\include"
 mvn clean install
 ```
+
+Contributing
```
Comment (fjuma) [inline README.md:136]: Just to maintain consistency with the Elytron repo, let's use Contributions here instead.

**Comment by cam-rod** [comment]
Submitted the changes

**Comment by fjuma** [comment]
Perfect, thanks @cam-rod! Just one more minor thing, it would be good to create and reference an SSLNTV JIRA issue here.

**Comment by cam-rod** [comment]
Alright, created issue [SSLNTV-30](https://issues.redhat.com/browse/SSLNTV-30).

**Pull request merged**